java/pom.xml

```diff
@@ -275,6 +275,10 @@
                 </module>
                 <module name="NewlineAtEndOfFile">
                 </module>
+                <module name="RegexpSingleline">
+                  <property name="format" value="\s+$"/>
+                  <property name="message" value="No trailing white space allowed."/>
+                </module>
```

Comment:
We need the following before `<checkstyleRules>`. Could you add that?
```xml
           <configuration>
+            <sourceDirectories>
+              <directory>${basedir}/src/java</directory>
+            </sourceDirectories>
```

Reply:
Thank you, I have updated the PR.